tests/crc32/main.c

```diff
@@ -82,11 +83,13 @@ perftest()
 			goto err1;
 		}
 
-		/* Find and print elasped time. */
-		delta = 1000000*((long long)(end.tv_sec - begin.tv_sec)) +
+		/* Find and print elasped time and speed. */
+		delta_us = 1000000*((long long)(end.tv_sec - begin.tv_sec)) +
 		    (end.tv_usec - begin.tv_usec);
-		printf("... in %zu blocks of size %zu:\t%.02f s\n",
-		    num_hashes, perfsizes[i], (double)delta / 1000000);
+		delta_s = (double)delta_us / 1000000.0;
+		printf("... in %zu blocks of size %zu:\t%.02f s\t%.01f MB/s\n",
+		    num_hashes, perfsizes[i], delta_s,
+		    ((double)bytes_to_hash / 1000000.0) / delta_s);
```

Comment:
I would have written this `(double)bytes_to_hash / delta_us` personally... MB/s is the same as B/us. :-)

Reply:
> MB/s is the same as B/us. :-)

Oh, I knew that.  However, when I experimented with showing GB/s, I lost track of all the `0`s.  I thought it was safer to treat each value separately.

... that said, now that I've defined `delta_us` instead of the previous version which was just `delta`, it's more clear.  Ok, I'll go with the assumption that any reader knows the connection between `M` and `u`.